utils/common/src/main/java/brooklyn/util/net/Networking.java

```diff
@@ -115,7 +113,7 @@ public static boolean isPortAvailable(InetAddress localAddress, int port) {
                     try {
                         ss.close();
                     } catch (IOException e) {
-                        /* should not be thrown */
+                        Exceptions.propagateIfFatal(e);
```

Comment:
worth keeping comment in any case that non-fatal problems closing can be ignored